ONSdigital/census-rm-case-processor
# Emitted messages not Zulu and event date not stamped

# Motivation and Context
Events weren't having the current date stamped onto them when they were being created. The messages being emitted to the Rabbit exchange had dates which were not Zulu (UTC) time (i.e. they lacked the trailing "Z").

# What has changed
Stamped the date. Formatted the messages going to Rabbit.

# How to test?
Load a sample. Check that the emitted messages to RH have dates with a "Z" on the end in UTC. Check the database to ensure that the date is being stamped onto the events.

# Links
Trello: https://trello.com/c/uPfisqE7

# Screenshots (if appropriate):
